src/webots/gui/WbNewWorldWizard.cpp

```diff
@@ -138,37 +137,37 @@ bool WbNewWorldWizard::validateCurrentPage() {
 }
 
 QString WbNewWorldWizard::fileName() const {
-  return mNameEdit->text();
+  return mWorldEdit->text();
 }
 
 QWizardPage *WbNewWorldWizard::createIntroPage() {
   QWizardPage *page = new QWizardPage(this);
-  page->setTitle(tr("New world creation"));
-  QLabel *label = new QLabel(tr("This wizard will help you creating a new world file."), page);
+  page->setTitle(introTitle());
+  QLabel *label = new QLabel(introText(), page);
   QVBoxLayout *layout = new QVBoxLayout(page);
   layout->addWidget(label);
   return page;
 }
 
-QWizardPage *WbNewWorldWizard::createFilePage() {
-  QWizardPage *page = new QWizardPage(this);
-  page->setTitle(tr("World file name"));
-  page->setSubTitle(tr("Please choose a file name for your new world file:"));
-  mNameEdit = new WbLineEdit(page);
-  QHBoxLayout *layout = new QHBoxLayout(page);
-  layout->addWidget(mNameEdit);
-  return page;
-}
-
 QWizardPage *WbNewWorldWizard::createWorldPage() {
   QWizardPage *page = new QWizardPage(this);
   page->setTitle(tr("World settings"));
-  page->setSubTitle(tr("Please select the features you want:"));
+  page->setSubTitle(tr("Please choose a name for the new world and select the features you want:"));
+  mWorldEdit = new WbLineEdit(page);
   mBackgroundCheckBox = new QCheckBox(page);
+  mBackgroundCheckBox->setChecked(true);
+  mBackgroundCheckBox->setText("Add a textured background");
```

Comment:
Don't we need to use `tr()` here as well (and for the other check box labels)?

Reply:
Yes, I will take care of fixing this.